Allow HTTP/2 max concurrent stream configuration (#8781)
traefik/traefik: docs/content/reference/static-configuration/cli-ref.md

`--entrypoints.<name>.http.tls.options`:  
Default TLS options for the routers linked to the entry point.

`--entrypoints.<name>.http2.maxconcurrentstreams`:  

> Specifies the number of concurrent streams that each client may have open at a time. (Default: ```250```)

Reviewer: ```suggestion
Specifies the number of concurrent streams per connection that each client is allowed to initiate. (Default: ```250```)
```
